src/index.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line quotes
```

Comment:
- [ ] One of the few issues left to fix includes eslint disabling. This one, in particular, can be avoided by using single quotes.